src/lib/Microsoft.Health.Fhir.Ingest/Telemetry/Metrics/IomtMetrics.cs

```diff
@@ -159,30 +151,14 @@ public static Metric FhirResourceSaved(ResourceType resourceType, ResourceOperat
         public static Metric UnhandledException(string exceptionName, string connectorStage)
         {
             EnsureArg.IsNotNull(exceptionName);
```

Comment:
Yes, in case of UnhandledException, since the metric name is 'UnhandledException' and the actual exception name will be the 'Name' dimension, I felt it makes sense to ensure its set so that the name dimension is set and we can know the exact exception. its being passed as the errorName arg which is optional and ToErrorMetric  doesn't enforce this value being set.
In the other cases, the metric name is the exception name as well which